dlt-connector/src/server/logger.ts

```diff
@@ -0,0 +1,11 @@
+import log4js from 'log4js'
+import CONFIG from '@/config'
+
+import { readFileSync } from 'fs'
+const options = JSON.parse(readFileSync(CONFIG.LOG4JS_CONFIG, 'utf-8'))
+
+log4js.configure(options)
+
+const logger = log4js.getLogger('dlt-connector')
```

Comment:
der Parameter für getLogger muss dem Category-namen in der log4js-configuration entsprechen. In deinem Fall hast du die Category in der log4js-config noch auf dht stehen.